src/controllers/sendResetToken.js

```diff
@@ -0,0 +1,114 @@
+import { randomBytes } from 'crypto';
+import models from '../db/models';
+import sendgrid from '../middlewares/helper';
+
+const { User } = models;
+
+/**
+  * @description class representing reset user password
+  *
+  * @class SendResettoken
+  */
+class SendResetToken {
+  /**
+    * @description - This method is responsible for creating creating password reset token
```

Comment:
Hi @okoroemeka, your description has double "creating", please take one out

Reply:
Thanks for pointing that out. I have taken care of it.

Reply:
Resolved. Thanks for pointing that out.